example/constants.ts.example

```diff
@@ -7,4 +7,4 @@
 export const defaultCity = {'id': 7, 'properties': {'name': 'Philadelphia','admin': 'PA'}};
 export const defaultScenario = 'RCP85';
 
-export const apiHost = 'https://api.staging.futurefeelslike.com';
+export const apiHost = 'https://staging.climate.azavea.com';
```

Comment:
Since people could theoretically check this project out, or fork it and make changes, and/or even just have access to the source code, let's update this to point to `climate.azavea.com` instead of staging since that's the production instance by default.